pkg/virt-operator/util/config_test.go

```diff
@@ -128,6 +128,43 @@ var _ = Describe("Operator Config", func() {
 			true),
 	)
 
+	Context("Read Shasums from image names", func() {
+		It("environment variables", func() {
```

Comment:
could you adjust the test description?
Right now the result is:
```
Operator Config Read Shasums from image names environment variables
```
which should be improved.
Probably deleting the `Context` with something like:
`It("should be able to extract shasum from image names")`

Reply:
Thanks! Done